### ISSUE TITLE
List.foldl to List.sum fix loses function input

### ISSUE THREAD

**Issue by bdukes** [jfmengels/elm-review-simplify#86]
Running `npx elm-review --template jfmengels/elm-review-simplify/example` with the following module:
```elm
module Example exposing (oneOrLessTrue)

oneOrLessTrue : List Bool -> Bool
oneOrLessTrue bools =
    (bools |> List.map boolToInt |> List.foldl (+) 0) >= 1

boolToInt : Bool -> Int
boolToInt bool =
    if bool then
        1
    else
        0
```

produces the following fix:

```diff
  -- ELM-REVIEW ERROR --------------------------------------- src/Example.elm:5:37
  
  Simplify: Use List.sum instead
  
  4| oneOrLessTrue bools =
  5|     (bools |> List.map boolToInt |> List.foldl (+) 0) >= 1
                                         ^^^^^^^^^^
  
  Using List.foldl (+) 0 is the same as using List.sum.
  
  I think I can fix this. Here is my proposal:
  
  4| oneOrLessTrue bools =
- 5|     (bools |> List.map boolToInt |> List.foldl (+) 0) >= 1
+ 6|     (List.sum) >= 1
  6|
  
  √ Do you wish to apply this fix? ... no
```

It looks like the input to `List.foldl (+) 0` is getting lost, instead of being fed into `List.sum`.

**Comment by lue-bird** [comment]
Thanks a lot for reporting

**Comment by jfmengels** [comment]
Released in v2.0.28
